stack/rest/src/test/java/org/apache/usergrid/rest/applications/collection/users/UserResourceIT.java

```diff
@@ -64,113 +64,58 @@
  * @author tnine
  */
 @Concurrent()
-public class UserResourceIT extends AbstractRestIT {
+public class UserResourceIT extends org.apache.usergrid.rest.test.resource2point0.AbstractRestIT {
```

Comment:
If you remove the old AbstractRestIT you wouldn't have to have to include the entire package name